src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import print_function
+import re
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.expect import pexpect_ssh
+from constants import ds_instance_name
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.defaultdebuglevel
+class TestDefaultDebugLevel(object):
+    """ Check sssd default debug level """
+    @pytest.mark.tier1_2
+    def test_0001_check_default_debug_level(self, multihost, backupsssdconf):
+        """
+        @Title: IDM-SSSD-TC: Check default debug level when sssd start
+        successfully
+        """
+        section = "domain/%s" % ds_instance_name
+        domain_params = {'debug_level': ''}
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf(section, domain_params, action='delete')
+        # stop sssd, delete sssd logs and cache, start sssd
+        tools.clear_sssd_cache()
+        log_list = ['sssd', 'sssd_example1', 'sssd_nss',
+                    'sssd_pam', 'sssd_implicit_files']
+        for log_filename in log_list:
+            log = '/var/log/sssd/%s.log' % log_filename
+            log_str = multihost.client[0].get_file_contents(log).decode(
+                'utf-8')
+            log_split = log_str.split("\n")
+            for index in range(len(log_split)-1):
+                log_single_line = log_split[index]
+                pattern = re.compile(r'\] \(0x\d*\): ')
+                debug_str = pattern.search(log_single_line).group()
+                # get debug_level in hex from log
+                value = debug_str[3:-3]
+                int_debug_level = int(value, 16)
+                # int('0x0040', 16) is 64
+                if int_debug_level > 64:
```

Comment:
You can compare `int_debug_level > 0x0040` directly. No need to convert 0x0040 to dec.